gallery/src/styles/NxCard/NxCardPage.tsx

```diff
@@ -37,31 +38,33 @@ const NxCardPage = () =>
             <td className="nx-cell"><code className="nx-code">.nx-card-container</code></td>
             <td className="nx-cell">Top-Level</td>
             <td className="nx-cell">
-              Basic card container class.
+              Container for <code className="nx-code">.nx-card</code>s
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card-container--row</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-card-container--no-wrap</code></td>
             <td className="nx-cell">Modifier of <code className="nx-code">.nx-card-container</code></td>
             <td className="nx-cell">
-              Container for cards that are displayed in row(s), both normal and variant layouts. Rows can be
-              transformed into columns by replacing this style with
-              <code className="nx-code">.nx-card-container--column</code> below.
+              By default if you put more cards into a container than can fit horizontally the cards will wrap into a
+              new row. This modifier is used when you do not want the cards to wrap. Use with care as this can break
+              layouts.
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card-container--column</code></td>
-            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card-container</code></td>
+            <td className="nx-cell"><code className="nx-code">.nx-card</code></td>
+            <td className="nx-cell">Element</td>
             <td className="nx-cell">
-              Container for cards that are displayed in a column. Cards displayed in a column can be transformed
-              into rows by replacing this style with <code className="nx-code">.nx-card-container--row</code> above.
+              Basic card element.
             </td>
           </tr>
           <tr className="nx-table-row">
-            <td className="nx-cell"><code className="nx-code">.nx-card</code></td>
-            <td className="nx-cell">Element</td>
+            <td className="nx-cell"><code className="nx-code">.nx-card--equal</code></td>
+            <td className="nx-cell">Modifier of <code className="nx-code">.nx-card</code></td>
             <td className="nx-cell">
-              Basic card element.
+              <code className="nx-code">.nx-card</code>s have a default <code className="nx-code">min-width</code>
```

Comment:
This class doesn't currently remove the `min-width`

Reply:
> This class doesn't currently remove the `min-width`

I did that on purpose. I didn't think that it was a good idea to let the cards get crunched below their minimum

Reply:
> I did that on purpose. I didn't think that it was a good idea to let the cards get crunched below their minimum

The way this description is written implies (to me) that both the min and max are removed by this class

Reply:
> > I did that on purpose. I didn't think that it was a good idea to let the cards get crunched below their minimum
> 
> The way this description is written implies (to me) that both the min and max are removed by this class

Re-written: https://github.com/sonatype/sonatype-react-shared-components/pull/307/commits/e8c4351e9005851c43777f251641c1bdbf6a112e